crates/host/src/wasmbus/mod.rs

```diff
@@ -1353,7 +1353,19 @@ impl Host {
                         };
                         if let Err(err) = timeout(
                             max_execution_time,
-                            actor.handle_invocation(context, params, result_subject, &transmitter),
+                            tokio::spawn({
+                                let transmitter = transmitter.clone();
+                                async move {
+                                    actor
+                                        .handle_invocation(
+                                            context,
+                                            params,
+                                            result_subject,
+                                            &transmitter,
+                                        )
+                                        .await
+                                }
+                            }),
```

Comment:
This fixes the issue that we had where an infinitely looping component would prevent other instances of that component from executing 👍🏻